src/test/resources/application-test.yml

```diff
@@ -1,12 +1,12 @@
 spring:
-  profiles:
-    include: stdout
-
   security:
     oauth2:
       resourceserver:
         jwt:
           public-key-location: classpath:local-public-key.pub
+  config:
```

Comment:
why this change?

Reply:
it was suggested by spring as the old config style is deprecated.